index.html

```diff
@@ -508,48 +552,58 @@ <h2 class="sign-up-block__title title">Get Start New Job</h2>
         </div>
       </section>
       <!-- Third section -->
-      <section class="third-section">
-        <div class="third-section__background">
-          <div class="container third-section__container">
+      <section class="post-job">
+        <div class="post-job__background">
+          <div class="container post-job__container">
             <!-- Third section left wrapper -->
-            <div class="third-section__left-wrapper">
+
+            <div class="post-job__video-block">
               <img
-                class="third-section__img"
-                src="./src/img/third-section-img.png"
+                class="post-job__img"
+                src="./src/img/post-job-img.png"
                 alt=""
               />
-              <a href="#" class="play-button"></a>
+
+              <button class="play-button">
```

Comment:
```suggestion
              <button class="button_play">
```

Notice that `.button_play {}` should contain only appearance properties of the button, positioning should be done with this selector `.video-block .button_play {}`